### PR TITLE
Set catkey to no refresh when updating with serial form.

### LINKED ISSUE #3687: When the digital serials form is submitted, mark the catkey as "do not refresh"
The main use case for the "Do not refresh" catkey setting are digital serials. These are items that have gone through the following steps:

- catkey entered
- metadata refreshed from Symphony
- serials metadata added in Argo
- catkey marked as "Do not refresh" (using the new form we're implementing right now in #3358 )

These items are then supposed to be marked "do not refresh" because a refresh from Symphony would overwrite the serials metadata added in Argo.

Given that we know that serials metadata should not have metadata automatically refreshed, we could eliminate the separate step of updating the catkey setting by automatically marking any catkey updated via the serials form with the "Do not refresh" attribute.

### PR DESCRIPTION
**Pull request by justinlittman**
closes #3687

## Why was this change made? 🤔
Less work for user.

## How was this change tested? 🤨

⚡ ⚠ If this change has cross service impact (including writing to shared file systems or interacting with other SDR APIs), ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test in [stage|qa] environment, in addition to specs. ⚡

⚡ ⚠ If this change updates the Argo UI, run all integration tests that use Argo and create a PR on the integration test repo to fix anything this change breaks. ⚡

### CODE REVIEW

**Comment by justinlittman** [comment]
Andrew approved.

File: spec/forms/serials_form_spec.rb [hunk 1]
```diff
@@ -321,5 +321,39 @@
...
+                structuredValue: [
+                  { value: 'factory DRO title', type: 'main title' }
+                ]
+              }
+            ],
+            purl: 'https://purl.stanford.edu/bc123df4567'
+          },
+          identification: {
+            catalogLinks: [
+              { catalog: 'symphony', refresh: false, catalogRecordId: '6671606' }
```
Comment (edsu) [inline spec/forms/serials_form_spec.rb:342]: It's kind of too bad all the boilerplate is needed to just test `refresh: false`?

File: spec/forms/serials_form_spec.rb [hunk 2]
```diff
@@ -321,5 +321,39 @@
         expect(object_client).to have_received(:update).with(params: expected)
       end
     end
+
+    context 'when catalog links' do
```
Comment (edsu) [inline spec/forms/serials_form_spec.rb:325]: Maybe 'when catalog links present' reads better?

**Review by edsu — APPROVED** [review]

File: spec/forms/serials_form_spec.rb (hunk 2 above)
Comment (justinlittman) [reply spec/forms/serials_form_spec.rb:325]: Changing.

**Pull request merged**